### PR TITLE
BD-51 feat: text area

### PR DESCRIPTION
**Pull request by KateTernovskaya**

### CODE REVIEW

File: src/components/text-area/TextArea.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            })}
+            htmlFor={id}
+          >
+            {label}
+          </label>
+        )}
+        <textarea
+          className={cn(
+            'custom-scrollbar focus:border-primary-700 w-full min-w-[284px] resize-none rounded-md border border-dark-100 bg-transparent p-2 text-sm text-light-900 outline-none transition-all scrollbar-thin focus:border-accent-700 focus:text-light-100 active:border-light-100 active:text-light-100 disabled:cursor-not-allowed disabled:text-dark-100',
+            {
```
Comment (sfgardev) [inline src/components/text-area/TextArea.tsx:29]: image

File: src/components/text-area/TextArea.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+export const Default: StoryObj<typeof meta> = {
+  args: {
+    defaultValue: 'Text area',
+    label: 'Text-Area',
+  },
+}
+
+export const Error: StoryObj<typeof meta> = {
+  args: {
+    defaultValue: 'Text area',
```
Comment (sfgardev) [inline src/components/text-area/TextArea.stories.tsx:25]: maybe add placeholder instead of defaultValue

**Review by sfgardev — CHANGES_REQUESTED** [review]
forgot to add re-export into src/index.ts

File: src/components/text-area/TextArea.stories.tsx (hunk 2 above)
Comment (KateTernovskaya) [reply src/components/text-area/TextArea.stories.tsx:25]: defaultValue was set to see text styles

**Review by sfgardev — APPROVED** [review]

**Pull request merged**